pytorch_lightning/plugins/precision/hpu_precision.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (C) 2021 Habana Labs, Ltd. an Intel Company
```

Comment:
keep the filename consistent. no other precision plugin implementations include `_precision` in the filename.